.github/ISSUE_TEMPLATE/01_BUG_REPORT.md

```diff
@@ -1,21 +0,0 @@
----
```

Comment:
I like the issue templates, it forces me (and others :) ) to wonder what context may be needed.